src/App.vue

```diff
@@ -263,20 +264,21 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
     return Boolean(sessionStorage.getItem(SessionStorageKeys.KeyCloakToken))
   }
 
-  /** The error to display in the fee summary component, if any. */
-  private get feeSummaryError (): string {
-    // TODO: finish this
-    // return '< Please complete required information'
```

Comment:
Is this case handled?

Reply:
Yep, line 274 in `App.vue`

Reply:
What about '< Please complete required information' ?

Reply:
The designs showed it as `You have unfinished changes` but i see what you're saying now. 
Different text dependant on summary mode i'm guessing?

Reply:
Updated in next commit 👍

Reply:
It's not just page-based. It depends whether required components have been touched or not. See details in 7145.